backend/ee/danswer/server/enterprise_settings/models.py

```diff
@@ -10,6 +18,9 @@ class EnterpriseSettings(BaseModel):
     use_custom_logo: bool = False
     use_custom_logotype: bool = False
 
+    # custom navigation
+    custom_nav_items: List[NavigationItem] = []
```

Comment:
I believe we should use something like `Field(default_factory=list)` here to prevent the same list from being shared.

https://docs.pydantic.dev/latest/concepts/fields/#default-values

Reply:
Doy!